test-data/unit/check-unreachable-code.test

```diff
@@ -841,6 +841,38 @@ def baz(x: int) -> int:
         return x
 [builtins fixtures/exception.pyi]
 
+[case testUnreachableFlagExcepts]
+# flags: --warn-unreachable
+from typing import Type, NoReturn, Tuple
+
+def error() -> NoReturn: ...
+
+ignore: Type[Exception]
+exclude: Tuple[Type[Exception], ...]
+class ContinueOnError: pass
```

Comment:
```suggestion
class ContinueOnError(Exception): pass
```

Is this what you meant? 🤔

Reply:
I wanted to provoke `self.check_except_handler_test(expr)` to produce and error and thus return `Any`. In this case, we must ignore it, otherwise `is_subtype` always evaluates to `True`.

I will add a test to make this more clear.